fanuc_lrmate200id_support/launch/robot_state_visualize_lrmate200id7l.launch

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<!--
+  Manipulator specific version of the state visualizer.
+
+  Defaults provided for LR Mate 200iD/7L:
+   - J23_factor = 1
+   - use_bswap = true
+   - 6 joints
+
+  Usage:
+    robot_state_visualize_lrmate200id7l.launch robot_ip:=<value>
+-->
+<launch>
+  <arg name="robot_ip" doc="IP of controller" />
+  <arg name="J23_factor" default="1" doc="Compensation factor for joint 2-3 coupling (-1, 0 or 1)" />
+  <arg name="use_bswap" default="true" doc="If true, robot driver will byte-swap all incoming and outgoing data" />
+
+  <rosparam command="load" file="$(find fanuc_lrmate200id_support)/config/joint_names_lrmate200id.yaml" />
```

Comment:
maybe this better be `joint_names_lrmate200id7l.yaml` since there is that file included

Reply:
Yes. Fixed.